bindata/bootkube/manifests/configmap-admin-kubeconfig-client-ca.yaml

```diff
@@ -3,6 +3,8 @@ kind: ConfigMap
 metadata:
   name: admin-kubeconfig-client-ca
   namespace: openshift-config
+  labels:
+    "auth.openshift.io/managed-certificate-type": ca-bundle
```

Comment:
danger!

Be really sure before you mess with this.  This data is gone forever if you change it.  @soltysh

Reply:
@deads2k i'm not going to change this, the label means the prometheus will pick this up and report the expiration for these certs.

Reply:
for inreplacable certs, we might want some special annotation maybe "openshift.io/do-not-touch-this"

Reply:
Why do we have to manually label those? Isn't that your controller job? If these are the initial certs, do we care?